### PR TITLE
add search page

### LINKED ISSUE #23: Create the search page including filters.
**Is your feature request related to a problem? Please describe.**
N/A

**Describe how the solution should behave**
Create the search page including all filters. The user should be able to see the filters modals and adjust the search accordingly.

**Screenshots of the design/if any**
image

**Additional context**
Ignore the notification modal another issue!

### PR DESCRIPTION
**Pull request by EsraaY21**
The design of the search page is done and the basic search is working but I haven't added the advanced search logic because I could not find any location value in the project details so I can't sort by the nearest nor can I find based on the location. Should we add it later when we have the data or should we remove it?
You can see the search page in `/search`

### CODE REVIEW

**Comment by aryankarim** [comment]
Don't worry about the functionality for now. This looks good! Good work! Though we need to have Arabic translations for them.

**Comment by EsraaY21** [comment]
Thank you. Okay, I will work on the translations.

**Comment by EsraaY21** [comment]
You can see the translation by going to `ar/search`

**Comment by aryankarim** [comment]
Good job Esraa! You can also change the direction from left to right when it is in Arabic. I will comment the code step by step. Its really easy.

File: components/Search/AdvancedSearch.js [hunk 1]
```diff
@@ -10,13 +10,18 @@ import {
   Divider,
   Button,
 } from "@chakra-ui/react";
+import { useTranslation } from "next-i18next";
+import { useRouter } from "next/dist/client/router";
```
Comment (aryanmajeed) [inline components/Search/AdvancedSearch.js:15]: import router like this: `import { useRouter as router } from "next/dist/client/router";`

File: components/Search/AdvancedSearch.js [hunk 2]
```diff
@@ -10,13 +10,18 @@ import {
...
 
 const AdvancedSearch = () => {
+  const { t } = useTranslation("search");
+  const { locale } = useRouter();
+
   return (
     <PopoverBody>
```
Comment (aryankarim) [inline components/Search/AdvancedSearch.js]: Only put this `dir={locale === "ar" ? "rtl" : "ltr"}` attribute on the `PopoverBody `component, and that will do the trick.

File: components/Search/Sort.js [hunk 3]
```diff
@@ -11,8 +11,13 @@ import {
...
 import { RiArrowDropDownLine } from "react-icons/ri";
+import { useTranslation } from "next-i18next";
+import { useRouter } from "next/dist/client/router";
 
 const Sort = () => {
+  const { t } = useTranslation("search");
+  const { locale } = useRouter();
+
   return (
     <Popover>
```
Comment (aryankarim) [inline components/Search/Sort.js:22]: you can add the same line `dir={locale === "ar" ? "rtl" : "ltr"}` for this `Popover` component. Let me know if that work.

Comment (IsraaY21) [reply components/Search/Sort.js:22]: Thank you! I will try that

**Review by MuhannedNoman — APPROVED** [review]
LGTM, feel free to merge.

**Comment by aryankarim** [comment]
I fixed the direction @EsraaY21. Pull the changes and try it out, then merge it if it works.

**Review by aryankarim — APPROVED** [review]

**Pull request merged**